cmd/create/machinepool/cmd.go

```diff
@@ -233,6 +248,52 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	// Single AZ machine pool for a multi-AZ cluster
+	var multiAZMachinePool bool
+	var availabilityZone string
+	if cluster.MultiAZ() {
```

Comment:
else {multiAZMachinePool=false}

Reply:
If I'm not wrong `multiAZMachinePool` is initialized to **false**